yt/yt/experiments/public/fio_plugin/plugin.cpp

```diff
@@ -0,0 +1,821 @@
+#include "plugin.hpp"
+
+#include <mutex>
```

Comment:
We use server -> ytlib -> client -> yt/yt/library -> core -> library -> util -> contrib -> stdlib pattern